### PR TITLE
Removed ENV PORT from all Docker files

### PR DESCRIPTION
**Pull request by hpandeycodeit**
Fixes remaining part of #1204 

## Proposed Changes
Removed the ENV PORT environment variable from all Docker files.

### CODE REVIEW

File: community/samples/serving/helloworld-elixir/Dockerfile [hunk 1]
```diff
@@ -31,7 +31,7 @@ RUN chown appuser /opt/app/var
 
 USER appuser
 
-ENV PORT=8080 MIX_ENV=prod REPLACE_OS_VARS=true
+MIX_ENV=prod REPLACE_OS_VARS=true
```
Comment (sebgoa) [inline community/samples/serving/helloworld-elixir/Dockerfile]: You delete the `ENV` key here

File: community/samples/serving/helloworld-elixir/README.md [hunk 2]
```diff
@@ -79,8 +79,7 @@ When asked, if you want to `Fetch and install dependencies? [Yn]` select `y`
    # Run everything else as 'appuser'
    USER appuser
 
-   # Document that the service listens on port 8080.
-   ENV PORT=8080 MIX_ENV=prod REPLACE_OS_VARS=true
+   MIX_ENV=prod REPLACE_OS_VARS=true
```
Comment (sebgoa) [inline community/samples/serving/helloworld-elixir/README.md]: same here

Comment (hpandeycodeit) [reply community/samples/serving/helloworld-elixir/README.md]: added the key at both the places. Thanks for pointing out!

**Comment by samodell** [comment]
/approve

Thanks @hpandeycodeit  and @sebgoa !

**Comment by hpandeycodeit** [comment]
Thank you @sebgoa @samodell!

**Pull request merged**